### ISSUE TITLE
Any interaction with an input field causes dropdown to close

### ISSUE THREAD

**Issue by navied** [trendmicro-frontend/react-dropdown#4]
I seem to be having some issues with implementing something similar to the "Search Filter" demo. When you click or interact with the input field it wants to close the dropdown. 

I tried downloading and running the example locally without the forced "open" and was able to reproduce the it locally with the example. Does the input field within the "Search Filter" only work when you are manually controlling when the dropdown will open or close?

**Comment by cheton** [comment]
You have to use **Dropdown.MenuWrapper** to wrap **Dropdown.Menu**. I've fixed the sample code in e6b672713115d74f52766cc0ffd4416629d22669.

```jsx
<Dropdown onToggle={() => {}}>
    <Dropdown.Toggle title="Sample" />
    <Dropdown.MenuWrapper>
        <SearchFilter />
        <Dropdown.Menu>
            <MenuItem>first item</MenuItem>
            <MenuItem>second item</MenuItem>
        </Dropdown.Menu>
    </Dropdown.MenuWrapper>
</Dropdown>
```

**Comment by navied** [comment]
Works great! Thank you for the speedy fix/response.
